packaging/shark-platform/pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "shark-platform"
+authors = [
+  {name = "SHARK Authors"},
+]
+description = "SHARK Platform meta package"
+readme = "README.md"
```

Comment:
Do you know which `README.md` this is? Should we use the repository root `/README.md`, instead of `/packaging/shark-platform/README.md`?

Reply:
For now it is the one side by side to the `pyproject.toml`, no idea if we can include the one in the repo root instead.